embeddings/pipeline/lightning_hps_pipeline.py

```diff
@@ -95,12 +109,18 @@ def _init_preprocessing_pipeline(self) -> None:
         else:
             self.preprocessing_pipeline = HuggingFacePreprocessingPipeline(
                 dataset_name=str(self.dataset_name_or_path),
-                persist_path=self.tmp_dataset_dir.name,
+                persist_path=str(self.dataset_path),
                 sample_missing_splits=(self.sample_dev_split_fraction, None),
                 ignore_test_subset=True,
                 load_dataset_kwargs=self.load_dataset_kwargs,
             )
 
+    def _init_evaluation_pipeline(
```

Comment:
I'd suggest changing name of this method to _get_evaluation_pipeline(...) since I think the `init` here should rather be used to initialize objects and variables and assign them as class attributes instead of returning them; e.g., the _init_preprocessing_pipeline method